core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/RepairGroup.java

```diff
@@ -124,35 +123,17 @@ public String toString()
     {
         Collection<RepairTask> tasks = new ArrayList<>();
 
-        boolean vnodeRepair = RepairOptions.RepairType.VNODE.equals(myRepairConfiguration.getRepairType());
-
         RepairTask.Builder builder = new RepairTask.Builder()
                 .withJMXProxyFactory(myJmxProxyFactory)
                 .withTableReference(myTableReference)
-                .withVnodeRepair(vnodeRepair)
                 .withTableRepairMetrics(myTableRepairMetrics)
                 .withRepairConfiguration(myRepairConfiguration);
 
-        if (vnodeRepair)
+        for (LongTokenRange range : myReplicaRepairGroup)
         {
-            for (LongTokenRange range : myReplicaRepairGroup)
-            {
-                builder.withTokenRanges(Collections.singletonList(range))
-                        .withReplicas(myReplicaRepairGroup.getReplicas());
-
-                tasks.add(builder.build());
-            }
-        }
-        else
-        {
-            Set<Host> replicas = myReplicaRepairGroup.getReplicas();
-
-            if (!replicas.isEmpty())
-            {
-                builder.withReplicas(replicas);
-            }
+            builder.withTokenRanges(Collections.singletonList(range))
+                    .withReplicas(myReplicaRepairGroup.getReplicas());
```

Comment:
Since the builder is reused within this for-loop and the withReplicas set the "same value" every time, I guess it can be moved up a few lines (outside the for-loop) - where the other "static" values are set to the builder...

Reply:
Good catch, I'll fix that.